x-pack/plugins/apm/public/components/shared/charts/ErrorRateChart/index.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiTitle } from '@elastic/eui';
+import theme from '@elastic/eui/dist/eui_theme_light.json';
+import { i18n } from '@kbn/i18n';
+import mean from 'lodash.mean';
+import React, { useCallback, useMemo } from 'react';
+import { useChartsSync } from '../../../../hooks/useChartsSync';
+import { useFetcher } from '../../../../hooks/useFetcher';
+import { useUrlParams } from '../../../../hooks/useUrlParams';
+import { callApmApi } from '../../../../services/rest/createCallApmApi';
+import { unit } from '../../../../style/variables';
+import { asPercent } from '../../../../utils/formatters';
+// @ts-ignore
+import CustomPlot from '../CustomPlot';
+
+const tickFormatY = (y?: number) => {
+  return asPercent(y || 0, 1);
+};
+
+export const ErrorRateChart = () => {
+  const { urlParams, uiFilters } = useUrlParams();
+  const syncedChartsProps = useChartsSync();
+
+  const { serviceName, start, end, errorGroupId } = urlParams;
+  const { data: errorRateData = [] } = useFetcher(() => {
+    if (serviceName && start && end) {
+      return callApmApi({
+        pathname: '/api/apm/services/{serviceName}/errors/rate',
+        params: {
+          path: {
+            serviceName,
+          },
+          query: {
+            start,
+            end,
+            uiFilters: JSON.stringify(uiFilters),
+            groupId: errorGroupId,
+          },
+        },
+      });
+    }
+  }, [serviceName, start, end, uiFilters, errorGroupId]);
+
+  const average = useMemo(() => {
+    return tickFormatY(mean(errorRateData.map((rate) => rate.y)));
```

Comment:
i see that we are using `tickFormatY` here to format the average value. Should we also use it in the other chart?

Reply:
In the distribution chart you mean?

Reply:
yes

Reply:
No, I only used `tickeFormatY` here to format the average to percentage. In distribution chart, the average is the number of errors created not percentage.